packages/rx_bloc_list/pubspec.yaml

```diff
@@ -1,24 +1,24 @@
 name: rx_bloc_list
 description: The rx_bloc_list package facilitates implementing infinity scroll and pull-to-refresh features with minimal setup. This package is meant to be used along with RxBloc ecosystem.
-version: 2.0.0
+version: 3.0.0
 homepage: https://www.primeholding.com
 repository: https://github.com/Prime-Holding/rx_bloc/tree/master/packages/rx_bloc_list
 issue_tracker: https://github.com/Prime-Holding/rx_bloc/issues
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.13.0 <3.0.0"
```

Comment:
@DDavidPrime, do we need to constraint the devs to use 2.13.0, or rx_bloc will be running fine with 2.12 as well?